cgi-bin/ljlib.pl

```diff
@@ -54,6 +54,14 @@ BEGIN
     }
 }
 
+# Now set up logging support for everybody else to access
+use Log::Log4perl;
```

Comment:
It wasn't on the checklist that we put together however many months ago. I'll add it. ✅